src/crimson/os/seastore/segment_cleaner.h

```diff
@@ -365,14 +384,22 @@ class SpaceTrackerSimple : public SpaceTrackerI {
   }
 
   int64_t get_usage(segment_id_t segment) const final {
-    return live_bytes_by_segment[segment];
+    return live_bytes_by_segment[segment].live_bytes;
+  }
+
+  double calc_utilization(segment_id_t segment) const final {
+    auto& seg_bytes = live_bytes_by_segment[segment];
+    if (!seg_bytes.live_bytes && !seg_bytes.total_bytes)
```

Comment:
Can seg_bytes.total_bytes actually be zero?

Reply:
yes, seg_bytes.total_bytes only count the space occupied by extents excluding record header and delta/extent infos. If a segment hasn't been holding any actual extents, its total_bytes would be zero.

Reply:
Or, maybe I should count the whole segment space instead of only those occupied by extents?